docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -123,6 +143,51 @@ spec:
         modes: ['moderator', 'observer']
 ```
 
+#### Joining sessions example
+
+Here is an example of Jeff with role `prod-access` connecting to
+a SSH server in the production environment.
+
+```code
+$ tsh ssh ubuntu@prod.teleport.example.com
+Teleport > Creating session with ID: 46e2af03-62d6-4e07-a886-43fe741ca044...
+Teleport > Controls
+  - CTRL-C: Leave the session
+  - t: Forcefully terminate the session (moderators only)
+Teleport > User jeff joined the session.
+Teleport > Waiting for required participants...
+```
+
+Jeff's session is paused, waiting for the required observers.
+
+Now Alice with the `auditor` role joins as a moderator and
+the session can begin. 
+
+```code
+$ tsh join --mode=moderator 46e2af03-62d6-4e07-a886-43fe741ca044
+Teleport > Creating session with ID: 46e2af03-62d6-4e07-a886-43fe741ca044...
+Teleport > Controls
+  - CTRL-C: Leave the session
+  - t: Forcefully terminate the session (moderators only)
+Teleport > User jeff joined the session.
+Teleport > Waiting for required participants...
+Teleport > User alice joined the session.
+Teleport > Connecting to prod.teleport.example.com over SSH
+
+ubuntu@prod.teleport.example.com % 
+```
+
+Here is an example of joining from the UI that is available for Server sessions.
+
+![Join Server Session from UI](../../../img/webui-active-session.png)
+
+<Admonition title="Note" type="tip">
+
+Joining as a moderator and observer is available for Servers in the UI Console.
```

Comment:
- "Servers" should be "servers", since it's not a proper noun
- What is the UI Console?

Reply:
This meant Web UI but it's redundant since we're showing the joining of servers.  Removed